rclcpp/wait_set/static_wait_set.cpp

```diff
@@ -0,0 +1,95 @@
+// Copyright 2021 Open Source Robotics Foundation, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <memory>
+#include <vector>
+
+#include "rclcpp/rclcpp.hpp"
+#include "std_msgs/msg/string.hpp"
+
+int main(int argc, char * argv[])
```

Comment:
Maybe this examples needs a comment explaining it like the others have.

Reply:
Also, this example also falls into that "not really an example" category, because it almost more of a demo showing each item works or a test.

I was thinking this would be more like `wait_set.cpp` but just with a static wait set.

Reply:
I created a static minimal subscriber version in the `minimal_subscriber` package.